react-site/src/App.js

```diff
@@ -0,0 +1,68 @@
+import './App.css';
+import ReactDOM from 'react-dom';
+import cambodia from './assets/cambodia-elephant.jpg';
+import me from './assets/ElizabethFrey.jpeg';
+import grad from './assets/IMG_7078.jpeg';
+
+function App(){
+  var name;
+  const buttonFunction = () =>{
+    console.log("The button has been clicked.");
```

Comment:
nice job including a button. And this was a good idea for trying to force something to appear for sure. There is no way that you would know this but in general you really shouldn't mess with the DOM, just because React has its own lifecycle and it is quite complicated and by forcing the DOM to render something, it messes with that lifecycle. Here it is fine, but in the future it gets messy. again you wouldnt know anything about this and it was definitely a good example of outside googling/research :)